perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -1200,13 +1237,49 @@ def ListConsistencyBenchmark(storage_schema, host_to_connect=None):
                final_result)
 
   # Final clean up: delete the objects still remaining.
-  DeleteObjectsConcurrently(storage_schema,
+  DeleteObjectsConcurrently(backend,
                             per_thread_objects_written,
-                            host_to_connect,
                             per_thread_objects_deleted)
   return final_result
 
 
+def TestBackend(backend):
```

Comment:
I've been working on it more, and I'm not sure that moving it is the right choice. It is definitely a test, but it's a test that needs the flags `azure_account`, `azure_key`, `bucket`, and `storage_provider` that are normally passed to `object_storage_api_tests.py`. I can make it into three separate unit tests, one for each provider, but the user will still have to provide the name of a test bucket they created for each service and their Azure account and key. I think it might be simpler to leave it in `object_storage_api_tests.py` for now. If we can automate the bucket setup, then I think it would make more sense to move it to the tests directory. What do you think?